elasticsearchdashboard/api.go

```diff
@@ -23,4 +23,6 @@ import (
 type EDClient interface {
 	GetHealthStatus() (*Health, error)
 	GetStateFromHealthResponse(health *Health) (esapi.DashboardServerState, error)
+	ExportDashboardSavedObjects() (*Response, error)
```

Comment:
Rename this methods to - `ExportSavedObjects` & `ImportSavedObjects`
They are implementing dashboard client interface anyway.